packages/device-utils/tests/modelUtils.test.ts

```diff
@@ -0,0 +1,35 @@
+import { DeviceModelInternal } from '@trezor/connect';
+
+import { pickByDeviceModel } from '../src';
+
+describe('pickByDeviceModel', () => {
+    const OPTIONS = {
+        default: 'default',
+        [DeviceModelInternal.T1B1]: null,
+        [DeviceModelInternal.T3T1]: 'T3T1',
+    };
+
+    const fixtures = [
+        {
+            description: 'should return correct value for a given device',
+            deviceModelInternal: DeviceModelInternal.T3T1,
+            expectedResult: 'T3T1',
+        },
+        {
+            description: 'should return null when the value for the picked device is null',
+            deviceModelInternal: DeviceModelInternal.T1B1,
+            expectedResult: null,
+        },
+        {
+            description: 'should return default when deviceModelInternal is undefined',
+            deviceModelInternal: undefined,
+            expectedResult: 'default',
+        },
+    ];
```

Comment:
fixtures should be in different file

Reply:
options can be part of fixtures, somehow

Reply:
18711f2b98c9e8493509db57009a79088ced546d